base/reducedim.jl

```diff
@@ -1231,6 +1231,150 @@ end
 
 reducedim1(R, A) = length(axes1(R)) == 1
 
+function findextrema!(f, op_mn, op_mx, Rval_mn, Rind_mn, Rval_mx, Rind_mx, A::AbstractArray{T,N}) where {T,N}
+    (isempty(Rval_mn) || isempty(Rval_mx) || isempty(A)) && return ((Rval_mn, Rind_mn), (Rval_mx, Rind_mx))
+    lsiz_mn = check_reducedims(Rval_mn, A)
+    lsiz_mx = check_reducedims(Rval_mx, A)
```

Comment:
This seems redundant. (We check `axes(Rval_mx) == axes(Rval_mn)` below.)

Reply:
There is indeed some redundancy. At most, `check_reducedims` on either `Rval_mn` or `Rval_mx` will suffice.